encrypt/encrypt.go

```diff
@@ -26,7 +26,8 @@ import (
 // Usage text that will be displayed as command line help text when using the
 // `help encrypt` command
 var Usage = `
-USAGE: %s encrypt -key <public-key-file> (-outdir <dir>) (-continue=true) [file(s)]
+USAGE: %s encrypt -key <public-key-file> (-target <target>) (-outdir <dir>) (-continue=true) [file(s)]
+
```

Comment:
```suggestion
```

Reply:
removed, not sure why github doesn't catch this